vendor/bat-native-ledger/src/bat/ledger/internal/test/mock_ledger_client.h

```diff
@@ -29,6 +29,14 @@ class MockLedgerClient : public ledger::LedgerClient {
 
   void CreateWallet(ledger::LedgerCallbackHandler* handler) override;
 
+  MOCK_METHOD6(LoadURL,
```

Comment:
why was this added here? we have `ledger_client_mock.h` where we define mocks

Reply:
Ah, that's confusing naming then :D - but, this appear to be the 'mock' in use by the code, as this change resolved the failing tests (related to the discussion we had on zoom).